docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -60,18 +60,52 @@ If a device is connected over Bluetooth or Wi-Fi and registers an application, S
 !!!
 
 
+###The below describes the behavior related to RSDL functionality only.   
+
+_**Changes**_   
+  1.	New appHMIType: REMOTE_CONTROL was added: all and any remote-control applications must register with this type.   
```

Comment:
Formatting fix needed, none of these numbered lists are indented on the dev portal [https://smartdevicelink.com/en/docs/pull_request/4e2a3eabeb434d53ab12e7a350c06aa602ccad86/basiccommunication/onappregistered/](url)